warzone/GameEngine.cpp

```diff
@@ -69,7 +69,7 @@ void GameEngine::startUpPhase() {
 		cout << "What map file would you like?";
 		cin >> mapFileName;
 		loader = new MapLoader(mapFileName);
-		if (loader->validateMapFormat() == true)
+		if (loader->validateMapFormatML() == true)
```

Comment:
Why rename this method?

Reply:
Did that so it's more clear, otherwise adapter, target, and adaptee would all have the same name methods which idk if they would like or not. Just did it to test that it was working but can change back if you think it's better